wps-common/src/main/java/au/org/aodn/aws/wps/status/StatusHelper.java

```diff
@@ -1,20 +1,179 @@
 package au.org.aodn.aws.wps.status;
 
+import net.opengis.ows._1.*;
+
+import net.opengis.wps._1_0.*;
+import net.opengis.ows._1.ObjectFactory;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+import javax.xml.datatype.XMLGregorianCalendar;
+import java.io.*;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.HashMap;
+import java.util.TimeZone;
+
 public class StatusHelper
 {
 
-    public static final String getStatusDocument(String s3Bucket, String statusFilename, EnumOperation operation, String jobId, EnumStatus jobStatus, String message, String messageCode)
+    static int CHUNK_SIZE = 512;
+
+    public static final String getStatusDocument(String s3Bucket, String statusFilename, String jobId, EnumStatus jobStatus, String message, String messageCode, HashMap<String, String> outputsHrefs)
     {
-        String statusDocument = null;
+        String statusLocation = getS3ExternalURL(s3Bucket, jobId + "/" + statusFilename);
+        ExecuteStatusBuilder statusBuilder = new ExecuteStatusBuilder(statusLocation, jobId);
+        return statusBuilder.createResponseDocument(jobStatus, message, messageCode, outputsHrefs);
```

Comment:
This is a bit convoluted - statushelper calls status builder calls status helper to build the document.  Perhaps move to StatusBuilder.

Reply:
Changed so the call to build the document is a call to ExecuteStatusBuilder - which then uses StatusHelper.